app/views/posts/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Showing User specific post</h1>
\ No newline at end of file
+<div class="post">
+  <div class="post-header">
+    <h2>Post #<%= @user.posts.index(@post) + 1 %>
+      by
+      <%= @user.name %></h2>
+    <h5>Comments:
+      <%= @post.comments_counter %>,&nbsp;Likes:
+      <%= @post.likes_counter %></h5>
+  </div>
+  <h3><%= @post.title %></h3>
+  <p><%= @post.text %></p>
+</div>
+
+<% if @post.comments_counter > 0 %>
+  <div class="comments">
+    <% @comments.each do |comment| %>
+      <li><%= comment.author.name %>:
+      <%= comment.text %></li>
+    <% end %>
+  </div>
+<%end%>
```

Comment:
- [ ] Kindly make sure the views of the app look like the provided wireframe

|Your website|Provided Wirframes|
|-|-|
|![Screenshot_20230629_003549](https://github.com/Odongo006/Rails_blog_App/assets/25227022/239ed7de-e178-4385-a668-193242be9765)|![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png)|